### PR TITLE
WIP - UI fixes

### PR DESCRIPTION
**Pull request by mahmoudfelfel**
This branch is for some UI fixes for the website on desktop and mobile.
I'm trying to follow exactly the designs as provided by Ahmed Galal.
It is a work in progress.

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32779

**Comment by sabeurthabti** [comment]
salam @mahmoudfelfel  - I am trying to remove the bill and anything bootstrap, have a look at : https://github.com/quran/audio.quran.com/pull/73 going to remove as much bootstrap as possible.

File: src/styles/bootstrap.overrides.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 @import 'fonts.scss';
 
+body {
+  font-family: "Montserrat-regular", sans-serif;
```
Comment (sabeurthabti) [inline src/styles/bootstrap.overrides.scss]: nice one,

**Comment by mahmoudfelfel** [comment]
Salam @sabeurthabti ok, if that PR got merged first I will update my branch.
I will close this PR and open a new one as I found the UI changes need some JS changes not only CSS, so I'm changing the branch name.

**Comment by mahmoudfelfel** [comment]
The new PR for UI fixes
https://github.com/quran/audio.quran.com/pull/77

**Pull request closed without merging**